### ISSUE TITLE
Implement route path helper

### ISSUE THREAD

**Issue by ChrisBr** [tongueroo/jets#308]
## Summary
I'm new to the Serverless world so I'm not 100% sure if the feature does make sense though 😄 

However, I'm wondering why route path helpers are missing? For instance

```ruby
resources :posts
```

should create path helpers like

```ruby
posts_path
...
```

## Motivation
One example for specs specs

```
get posts_path

expect(response).to be_successfull
```

## Guide-level explanation

See Rails documentation: https://guides.rubyonrails.org/routing.html#path-and-url-helpers

## Unresolved Questions

Do path helper make sense in the context of Serverless or is this feature intentionally missing?

**Comment by tongueroo** [comment]
Yes. Think this makes a lot of sense. One of the things that have been working in my spare time. Think will be a major improvement and really want this, so it's just a matter of time. Unsure when will be done.

**Comment by tongueroo** [comment]
Handle in #318 Released in [v2.0.0](https://github.com/tongueroo/jets/blob/master/CHANGELOG.md) 🎉

**Comment by ChrisBr** [comment]
Can we make this available in tests? Or at least add some documentation?

```ruby
Jets.application.routes.draw do
  resources :recipes, only: [:index]
end
```

```ruby
require 'spec_helper'

RSpec.describe "Searches", type: :request do
  describe "GET /recipes" do
    it "works" do
      get recipes_path, query: { query: :query }

      expect_json_response('result/').not_to be_empty
    end
  end
end
```

Will throw a:
> undefined local variable or method `recipes_path' for 
#<RSpec::ExampleGroups::Searches::GETRecipes:0x00007f8dd45d4490>

Also there is no documentation for it here https://rubyonjets.com/docs/routing/

Thanks anyway for the quick implementation!

**Comment by tongueroo** [comment]
> RE: Can we make this available in tests? 

Done in #332 Released in [v2.0.4](https://github.com/tongueroo/jets/blob/master/CHANGELOG.md)
